test/parallel/test-console-is-a-namespace.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+// https://heycam.github.io/webidl/#es-namespaces
+// https://console.spec.whatwg.org/#console-namespace
+// https://github.com/w3c/web-platform-tests/blob/master/console/console-is-a-namespace.any.js
+
+const common = require('../common');
+const assert = require('assert');
+const { test, assert_equals, assert_true, assert_false } = common.WPT;
+
+assert.doesNotThrow(() => {
```

Comment:
Oh you're right! It's better to keep the test case to verify the update, and we don't need to move it into a separate file since this test is related to namespace :)